parent-mojo/pom.xml

```diff
@@ -108,8 +108,8 @@
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.8.1</version>
           <configuration>
-            <source>11</source>
-            <target>11</target>
+            <source>8</source>
+            <target>8</target>
```

Comment:
This, and the equivalent change in the root pom file, is the specific change that restores support. Java 8 should be used for compiling as long as we want to keep Java 8 support.